Authenticators/Elkarte/elkarteauth.py

```diff
@@ -0,0 +1,887 @@
+#!/usr/bin/env python
+# -*- coding: utf-8
+
+# Elkarte modification Copyright (C) 2015 Antony Derham <antony@xydre.com>
```

Comment:
I honestly didn't know how to go about this. I'll make the changes you've suggested.